corehq/apps/smsbillables/migrations/0007_smsbillable_multipart_count.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0006_remove_smsbillable_api_response'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
i think the update will be just as slow as setting the default. 3 million rows isn't too much, but i agree it's enough that we probably don't want to slow down a deploy for it.  since we're just adding a field we could also probably run this migration before deploy